lib/compute/index.js

```diff
@@ -50,17 +64,17 @@ exports.image = function image(id, payload) {
         return reject(err);
       }
       var buf = Buffer.concat(arr);
-      imageCc.enqueue({
-        id: id,
-        payload: buf
-      }, function(err, res) {
-        if (err) {
-          logger.error('process.error', err);
-          reject(AppError.processingError(err));
-        } else {
-          resolve(res);
-        }
-      });
+
+      resolve(P.all(Object.keys(SIZES).map(function(variant) {
+        var hw = SIZES[variant];
```

Comment:
nit: `size` or even `sz` might be a clearer name here than `hw`, which to me reads either "hardware" or "high-water" on first glance.